tests/chainer_tests/functions_tests/loss_tests/test_huber_loss.py

```diff
@@ -15,15 +15,15 @@
     [{'dtype': numpy.float16,
       'forward_options': {'rtol': 5e-3, 'atol': 5e-3},
       'backward_options': {'eps': 1e-1, 'rtol': 1e-1, 'atol': 1e-1},
-      'double_backward_options': {'eps': 1e-1, 'rtol': 1e-1, 'atol': 1e-1}},
+      'double_backward_options': {'eps': 1e-2, 'rtol': 1e-2, 'atol': 1e-2}},
```

Comment:
With the large shape `(2, 5, 3, 3)` it causes round-off errors.
https://github.com/chainer/chainer/blob/f57e75ea0e7145c8a5c9399f9380ffa92336405e/chainer/gradient_check.py#L557-L567

How about setting `dtype=float64` for `check_backward` and `check_double_backward`?
```diff
         gradient_check.check_backward(
-            f, (x_data, t_data), y_grad, **self.backward_options)
+            f, (x_data, t_data), y_grad,
+            dtype=numpy.float64, **self.backward_options)
```